README.md

```diff
@@ -1,9 +1,33 @@
-[![Review Assignment Due Date](https://classroom.github.com/assets/deadline-readme-button-24ddc0f5d75046c5622901739e7c5dd533143b0c8e959d652212380cedb1ea36.svg)](https://classroom.github.com/a/HUOoSZXh)
-# Assignment 1 (R group)
+library(tidyverse)
+library(tinytex)
+library(rmarkdown)
+library(palmerpenguins)
+library(dplyr)
+library(tibble)
```

Comment:
No need to include libraries for ```dplyr``` and ```tibble``` as both packages are already in tidyverse.